geoips/commandline/ancillary_info/cmd_instructions.yaml

```diff
@@ -233,8 +233,8 @@ instructions:
   geoips_list_plugins:
     help_str: |
       List off plugins found under all, or a specified GeoIPS package. This information
-      can then be used with `geoips get-plugin <interface_name> <plugin_name>` to get
-      more information about a certain plugin.
+      can then be used with `geoips describe-plugin <interface_name> <plugin_name>` to
+      describe more information about a certain plugin.
```

Comment:
Is "describe" appropriate here?

Reply:
Fixed this as well. Find and replace mistakes here.